src/App.js

```diff
@@ -7,27 +7,28 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import CommentList from "./components/CommentList";
 
 function App() {
-  const [currentArticleID, setCurrentArticleID] = useState(3);
+  const [currentArticleID, setCurrentArticleID] = useState(0);
```

Comment:
Is there a reason you have this in state? `SingleArticle` and `CommentList` have the article_id from the path `/article/:article_id` and can get it using `useParams`. This seems redundant and will only be a second source of truth which should be removed.